### PR TITLE
Adding JFrog Xray Violations and Logs Integrations

### PR DESCRIPTION
**Pull request by MahithaB**
### What does this PR do?

Adding Xray Logs and Violations Dashboards

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: jfrog_platform/README.md [hunk 1]
```diff
@@ -29,44 +31,97 @@ JFrog Artifactory’s/Xray's Metrics API integration with Datadog allows you to
...
-
-    Adding proper metadata is the key to unlocking the full potential of your logs in Datadog. By default, the hostname and timestamp fields should be remapped.
 
+1. Fluentd installation :
+
+    Please follow the detailed steps for Fluentd instllation based on your installation type and defining the environment variable correctly [here][18]  
+ 
+2. Fluentd configuration : 
+        
+    #### Artifactory Configuration
```
Comment (apigirl) [inline jfrog_platform/README.md:44]:
```suggestion
   **Artifactory Configuration**
```
using a h4 in an ordered list won't render, so I suggest updating it to bold

File: jfrog_platform/README.md [hunk 2]
```diff
@@ -29,44 +31,97 @@ JFrog Artifactory’s/Xray's Metrics API integration with Datadog allows you to
...
+    Download the artifactory fluentd configuration file to a directory the user has permissions to write, such as the $JF_PRODUCT_DATA_INTERNAL locations discussed above in the [Environment Configuration](#environment-configuration) section.
```
Comment (apigirl) [inline jfrog_platform/README.md:45]:
```suggestion
    Download the artifactory fluentd configuration file to a directory the user has permissions to write, such as the `$JF_PRODUCT_DATA_INTERNAL` locations discussed in the [Environment Configuration](#environment-configuration) section.
```

File: jfrog_platform/README.md [hunk 3]
```diff
@@ -29,44 +31,97 @@ JFrog Artifactory’s/Xray's Metrics API integration with Datadog allows you to
...
+    
+    ````text
+    cd $JF_PRODUCT_DATA_INTERNAL
+    wget https://raw.githubusercontent.com/jfrog/log-analytics-datadog/master/fluent.conf.rt
+    ````
+    
+    Override the match directive(last section) of the downloaded `fluent.conf.rt` with the details given below
```
Comment (apigirl) [inline jfrog_platform/README.md:52]:
```suggestion
    Override the match directive (last section) of the downloaded `fluent.conf.rt` with the details given below:
```

File: jfrog_platform/README.md [hunk 4]
```diff
@@ -29,44 +31,97 @@ JFrog Artifactory’s/Xray's Metrics API integration with Datadog allows you to
...
+    #### Xray Configuration
+    
+    Download the Xray fluentd configuration file to a directory the user has permissions to write, such as the $JF_PRODUCT_DATA_INTERNAL locations discussed above in the [Environment Configuration](#environment-configuration) section.
+    
+    ````text
+    cd $JF_PRODUCT_DATA_INTERNAL
+    wget https://raw.githubusercontent.com/jfrog/log-analytics-datadog/master/fluent.conf.xray
+    ````
+    
+    Fill in the JPD_URL, USER, JFROG_API_KEY fields in the source directive of the downloaded `fluent.conf.xray` with the details given below
```
Comment (apigirl) [inline jfrog_platform/README.md:79]:
```suggestion
    Fill in the `JPD_URL`, `USER`, `JFROG_API_KEY` fields in the source directive of the downloaded `fluent.conf.xray` with the details given below:
```

File: jfrog_platform/README.md (hunk 1 above)
Comment (apigirl) [reply jfrog_platform/README.md:44]: It also might be good to break this section into a, b, c sub steps

File: jfrog_platform/README.md [hunk 5]
```diff
@@ -29,44 +31,97 @@ JFrog Artifactory’s/Xray's Metrics API integration with Datadog allows you to
...
-
-    To enable this integration, run the td-agent on `artifactory` pods:
+    To enable this integration, run the td-agent on `artifactory`\ `xray` instances:
 
     ``` 
     td-agent
     ```
 
-    The API key is configured in `td-agent`, which will start sending logs to Datadog. 
+    The API key is configured in `td-agent`, which will start sending logs to Datadog. For other types of installation, please refer to the [original documentation][18]
```
Comment (apigirl) [inline jfrog_platform/README.md:124]:
```suggestion
    The API key is configured in `td-agent`, which will start sending logs to Datadog. For other types of installation, see the [jFrog documentation][18]
```

File: jfrog_platform/assets/dashboards/xray_logs.json [hunk 6]
```diff
@@ -0,0 +1,280 @@
+{
+    "author_name": "Datadog",
+    "description": "This dashboard provides a summary of access, service, and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight\n\nSource code is also available in  [Github](https://github.com/jfrog/log-analytics-datadog)\n\n",
```
Comment (apigirl) [inline jfrog_platform/assets/dashboards/xray_logs.json:3]:
```suggestion
    "description": "This dashboard provides a summary of access, service, and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight\n\nSource code is also available in [Github](https://github.com/jfrog/log-analytics-datadog)\n\n",
```

**Review by apigirl — COMMENTED** [review]
docs feedback

**Comment by MahithaB** [comment]
@apigirl I've changed the README and xray_logs.json based on the suggestions. Can you please review it again. Thanks!

**Review by hithwen — APPROVED** [review]

**Pull request merged**